dev/com.ibm.ws.app.manager.war/resources/OSGI-INF/l10n/metatype.properties

```
@@ -31,4 +31,9 @@ classloader.desc=Defines the settings for the application classloader.
 
 appBnd.name=Application Bindings
 appBnd.name$Ref=Application bindings reference
-appBnd.desc=Binds general deployment information included in the application to specific resources.
\ No newline at end of file
+appBnd.desc=Binds general deployment information included in the application to specific resources.
+
+earapp.addWebModuleClassPathTo.name=The class loader used for Web Application class paths
+earapp.addWebModuleClassPathTo.desc=Defines the class loader used for JARs referenced by Web Application Class-Path headers.
+earapp.addWebModuleClassPathTo.warLoader=Use the Web Application class loader for the JARs referenced by a Web Application Class-Path header.
+earapp.addWebModuleClassPathTo.earLoader=Use the parent Enterprise Application class loader for the JARs referenced by a Web Application Class-Path header.
```

```suggestion
earapp.addWebModuleClassPathTo.earLoader=Use the parent enterprise application class loader for the JARs that are referenced by a web application class-path header.
```

similar question as the previous